_data-prepper/common-use-cases/event-aggregation.md

```diff
@@ -0,0 +1,135 @@
+---
+layout: default
+title: Event aggregation with Data Prepper
+parent: Common use cases
+nav_order: 40
+---
+
+# Event aggregation with Data Prepper
+
+You can use Data Prepper to aggregate data from different events over a period of time. Aggregating events can help reduce unnecessary log volume and handle use cases like multiline logs that come in as separate events. The [Aggregate processor]({{site.url}}{{site.baseurl}}/data-prepper/pipelines/configuration/processors/aggregate/) is a stateful processor that groups events based on the values for a set of specified identification keys, and performs a configurable action on each group.
```

Comment:
I think it may be advantageous to format the aggregate processor as [`aggregate` processor](https://opensearch.org/docs/latest/data-prepper/pipelines/configuration/processors/aggregate/) or [`aggregate`](https://opensearch.org/docs/latest/data-prepper/pipelines/configuration/processors/aggregate/) processor.

Thus, change:

```
The [Aggregate processor]({{site.url}}{{site.baseurl}}/data-prepper/pipelines/configuration/processors/aggregate/)
```

to

```
The [`aggregate` processor]({{site.url}}{{site.baseurl}}/data-prepper/pipelines/configuration/processors/aggregate/)
```